docs/source/llms/prompt-engineering/index.rst

```diff
@@ -20,27 +20,27 @@ Quickstart
 
 The following guide will get you started with MLflow's UI for prompt engineering.
 
-Step 1: Create an MLflow Deployments Server Completions or Chat Endpoint
+Step 1: Create an MLflow AI Gateway Completions or Chat Endpoint
 ------------------------------------------------------------------------
-To use the prompt engineering UI, you need to create one or more :ref:`MLflow Deployments Server <deployments>`
+To use the prompt engineering UI, you need to create one or more :ref:`MLflow AI Gateway <deployments>`
 completions or chat :ref:`Endpoints <deployments-endpoints>`. Follow the
-:ref:`MLflow Deployments Server Quickstart guide <deployments-quickstart>` to easily create an endpoint in less than five
-minutes. If you already have access to an MLflow Deployments Server endpoint of type ``llm/v1/completions``
+:ref:`MLflow AI Gateway Quickstart guide <deployments-quickstart>` to easily create an endpoint in less than five
```

Comment:
Are we renaming the alias hyperlink?

Reply:
Do we need to?

Reply:
we don't need to. That was just a confirmation to make sure it was intentional to leave it :)